packages/compass-e2e-tests/helpers/commands/add-collection.ts

```diff
@@ -40,10 +40,10 @@ export async function addCollection(
   const createModalElement = await browser.$(Selectors.CreateCollectionModal);
   await createModalElement.waitForDisplayed();
 
-  const collectionInput = await browser.$(
-    Selectors.CreateDatabaseCollectionName
+  await browser.setValueVisible(
```

Comment:
browser.setValue() is a webdriverio command that basically does browser.clearValue() followed by browser.addValue(). browser.clearValue() is unreliably (ie. often does nothing) when testing an electron app. Most of the time we're actually only adding text to an empty input so the clear part is irrelevant, but we had almost 50 cases of setValue() and I did NOT feel like testing each one individually especially because many are in commands and then those commands may or may not act on an empty value..

The other benefit of setValueVisible() is that it is our command and we have extra useful debugging in there and it automatically retries, etc. ie. all the first things you'll try if a test fails on setValue() anyway.